plugins/outputs/wavefront/wavefront_test.go

```diff
@@ -264,6 +269,7 @@ func TestBuildValueString(t *testing.T) {
 
 }
 
+/*
```

Comment:
Delete or update this test as appropriate.

Reply:
Should TestBuildValueString should remain since the functionality is deprecated but not removed yet?

Did delete commented out test below (TestFormatMetricPoint)

Reply:
We can remove TestBuildValueString once the option is removed, for now its best to leave it.